lib/http_router.ex

```diff
@@ -395,20 +448,20 @@ defmodule HttpRouter do
   defp build_body(handler, action, caller), do: build_body(handler, action, caller, :skip)
   defp build_body(handler, action, _caller, add_header) do
     header = case add_header do
-        :json -> [{"accept", "application/json"}]
-        :xml  -> [{"accept", "application/xml"}]
+        # :json -> [{"accept", "application/json"}]
+        # :xml  -> [{"accept", "application/xml"}]
```

Comment:
These weren't used really anyways, but I've commented them here (as opposed to removing) because I'm circling back to actually implement this in the near future.